### DISCUSSION TITLE
Make game result visible in battlesnake Recent Games

### DISCUSSION THREAD

**Discussion by mawilson** [BattlesnakeOfficial/feedback#156] [Feature Requests]
It would be nice if a battlesnake's Recent Games showings had an option to view the result (win, loss, or tie) of said games. I don't often view the battlesnake UI's Recent Games list, because it doesn't show me results without clicking & watching the game.

I can see this being something that should be optional, since maybe you would prefer not to be spoiled of a game's outcome before clicking on a game. The UI could have a toggle. That toggle could be set for the user, or could just live on the page, so every page load it would be false by default, but once toggled on, results would be shown.

MSPaint mockup for reference
![recentGamesResults_mockup](https://user-images.githubusercontent.com/2830497/148302405-16b5c428-4865-40a6-bba0-32609cb5c8e2.png)

**Comment by md-hexdrive** [comment]
Good Idea!
